jobs/github-event/check_suite/completed.js

```diff
@@ -0,0 +1,19 @@
+/*
+
+jobs/github-event/check_run/completed.js
+
+Receives webhook events for when a check suite is completed.
+
+*/
+
+const _ = require('lodash')
+const onBranchStatus = require('../../../lib/on-branch-status')
+
+module.exports = async function ({ status, conclusion, head_sha, repository, installation }) { // eslint-disable-line
+  // This shouldn’t be possible, since this is the completed event handler, but hey.
```

Comment:
have you seen this be any different? Sounds redundant otherwise?

Reply:
No, but it’s a failsafe in case GitHub’s event payload and the actual state aren’t synced up yet/anymore. You never know.